website/docs/r/vapp_vm.html.markdown

```diff
@@ -208,6 +208,24 @@ resource "vcd_vapp_vm" "emptyVM" {
 
 ```
 
+## Example Usage (vApp template with multi VMs)
+This example shows how to create a VM using vApp template with multi VMs.
```

Comment:
New line seems to be missing between header and this sentence. Also, suggest clarifying more:

```suggestion

This example shows how to create a VM from a vApp template with multiple VMs by specifying which VM to use.
```

Reply:
updated, also there isn't need for new line as Paragraph notions are there and also this inline with other examples.